src/Matej.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej;
+
+class Matej
+{
+    /** @var string */
+    private $clientId;
+    /** @var string */
+    private $apiKey;
+
+    /**
```

Comment:
v PHP 7 bych se anotacim vyhnul, pokud mas typovou kontrolu primo v metode, je to redundantni informace

Reply:
Jo, urcite, dam pryc. A teda doufame, ze pri ty transpilaci se tam zase budou generovat :)?